utils/constants.py

```diff
@@ -3,11 +3,13 @@
 DAMAGE_TYPES = ('acid', 'bludgeoning', 'cold', 'fire', 'force', 'lightning', 'necrotic', 'piercing', 'poison',
                 'psychic', 'radiant', 'slashing', 'thunder')
 
+ABILITY_NAMES = ('prof_bonus','strength', 'dexterity', 'constitution', 'intelligence', 'wisdom', 'charisma')
+
 STAT_NAMES = ('strength', 'dexterity', 'constitution', 'intelligence', 'wisdom', 'charisma')
 
-STAT_ABBREVIATIONS = ('str', 'dex', 'con', 'int', 'wis', 'cha')
+STAT_ABBREVIATIONS = ('pro','str', 'dex', 'con', 'int', 'wis', 'cha')
 
-STAT_ABBR_MAP = {'str': 'Strength', 'dex': 'Dexterity', 'con': 'Constitution',
+STAT_ABBR_MAP = {'pro': 'Proficiency Bonus','str': 'Strength', 'dex': 'Dexterity', 'con': 'Constitution',
```

Comment:
So this would also affect anything that checks against these lists, such as the `-with <stat>` argument for `!cast`. Is that intentional?

Reply:
Would also allow `!save pro` to make a Proficiency Bonus save

Reply:
And potentially break this if I'm not mistaken

https://github.com/avrae/avrae/blob/992fa51e7e5e4a9bab7780478e96eb81a2c0f2ba/cogs5e/sheets/beyond.py#L173

Reply:
You would be correct about that breaking all of those things, which is why I'm just going to manually add proficiency when it is needed.